### DISCUSSION TITLE
Blog posts with an image.

### DISCUSSION THREAD

**Discussion by bajun65537** [jitinnair1/gradfolio#15] [Q&A] [answered]
I am having hard time understanding where to include images for blog posts. Say I have a png in `_posts` folder and wish to include it in the blog post what should be a relative path to that image?

**Comment by jitinnair1** [comment, marked as answer]
Jekyll by default doesn’t copy assets placed in the _posts folder. You’ll need to place it in another folder (not something with an _ underscore). Create `/assets/images/` and place the image there. But then, Jekyll will look for the image within the virtual folder hierarchy, with the year, month and date all represented by folders. So, the image has to be referred, relative to the root directory by adding `../../../` before `/assets/images/`.

So, your final relative URL will become `![](../../../assets/images/img.png)`

**Reply by bajun65537** [reply]
Fair. Problem solved. Thanks!
